cleo_plugins/DebugUtils/DebugUtils.cpp

```diff
@@ -183,7 +183,7 @@ class DebugUtils
 
     static bool WINAPI OnScriptProcess(CScriptThread* thread)
     {
-        currScript.Reset();
+        currScript.Begin(thread);
```

Comment:
this line can be removed. if the script is different we call `Begin` anyway (in ProcessCommand)

Reply:
Yes but it make sense from general design viewpoint. 
Logic inside ProcessCommand is just patching weird behavior.